src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir-book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: "Avenir-heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  color: #3d4e61;
+  font-family: Avenir-book, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  letter-spacing: 0;
+  text-align: left;
+  font-weight: 300;
```

Comment:
all these styles shouldn't be here. It's a bad idea to add these styles to the `*` selector